Notifications/notifications_actions_customization.md

```diff
@@ -12,48 +12,51 @@ This document is a starting point for engaging the community and standards bodie
 Native applications that also have a Web counterpart might want the behavior of latter to be as similar to the former as much as possible to provide a consistent user experience across various devices and platforms. VoIP (Voice over Internet Protocol) applications - e.g., Microsoft Teams, Google Meet, Zoom, etc - are no exception and have become increasingly popular and demanded for remote collaboration over the last couple of years.
 
 ![call_teams_notification](https://user-images.githubusercontent.com/80070607/180506585-73f05e29-5676-4686-88dc-bd2fa05b8e6d.png)  
+
 *Figure 1: Calling notification scenario.*
 
-We would like to propose an extension to the [Notifications API](https://notifications.spec.whatwg.org/) standard for incoming call scenarios to allow the notification action buttons to be customized and also allow the application to play a custom ringtone. This capability would make the incoming call notifications, which may require a faster immediate response, clearly distinguishable from the others to the user and would also contribute to increasing accessibility on the Web. Moreover, apps would still be able to use their trademark ringtones on the web and users would be capable of identifying the calling app just by listening to it.
+We would like to propose an extension to the [Notifications API](https://notifications.spec.whatwg.org/) standard for incoming call scenarios to allow the notification action buttons to be customized and also allow the application to play a ringtone. This capability would make the incoming call notifications, which may require a faster immediate response, clearly distinguishable from the others to the user and would also contribute to increasing accessibility on the Web.
```

Comment:
Should we also say something about priority break through and increased priorities for the incoming call scenario?

Reply:
I added a paragraph in the `Proposed Solution` section about it. What do you think?

Reply:
I think we should also put something in the motivation section here.  Maybe something like:  Browsers may also decide to increase the priority of the toast notification for the incoming call scenario by making the toast notification show immediately and for a longer duration.

Reply:
Added the mention to priority breakthrough and increased priority in the motivation section. Thanks!